### DISCUSSION TITLE
How to replace Source path with c# string

### DISCUSSION THREAD

**Discussion by blackuGT** [AvaloniaUI/Avalonia#6036] [Q&A] [answered]
Hi! I want to make sth like image slider on left side of my window.

All what I need (and don't know how to do it) is replace Source path and change the image every few seconds. As for changing path, I mean something like this:

`<Image Source="/Assets/theme.jpg" MaxWidth="400" MaxHeight="600" DockPanel.Dock="Left"></Image>`

`<Image Source="{Source}" MaxWidth="400" MaxHeight="600" DockPanel.Dock="Left"></Image>`

**Comment by maxkatz6** [comment, marked as answer]
You need to create a converter, that will create instance of Bitmap class from your string.

**Reply by maxkatz6** [reply]
See https://github.com/AvaloniaUI/Avalonia/issues/3774
